pom.xml

```diff
@@ -509,7 +509,7 @@
             <dependency>
                 <groupId>io.prestosql.hadoop</groupId>
                 <artifactId>hadoop-apache</artifactId>
-                <version>3.2.0-9</version>
+                <version>3.2.0-10-20201208.205558-2</version>
```

Comment:
what is `205558-2` version part?

Reply:
This is the snapshot I deployed so the CI could run. I forgot to put up the dependent PR: https://github.com/prestosql/presto-hadoop-apache/pull/25

Reply:
Thanks. The other one looks good.

Reply:
Curious. What are the benefits of this PR? Apart from obvious fact that it is cleaner as things are placed more correctly?

Reply:
It's cleaner and makes it easier to see what's going on.